PancakeSwap-Bots/Large-swap/src/data.fetcher.spec.ts

```diff
@@ -0,0 +1,108 @@
+import DataFetcher from "./data.fetcher";
+import { BigNumber, utils } from "ethers";
+import { createAddress, MockEthersProvider } from "forta-agent-tools/lib/tests";
+import { PANCAKE_FACTORY_ADDRESS, INIT_CODE_PAIR_HASH, PANCAKE_PAIR_ABI, ERC20ABI } from "./constants";
+
+//pair, token0, token1, block
+const PAIRS: any[][] = [
+  [
+    "0xb3db622eeca1dd7aae380bfa213b5728945465e6",
+    "0x0d420c4f278b70fc62906f8898d7e04dbe601574",
+    "0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c",
+    1001,
+  ],
+  [
+    "0xeaa54b28fb10735b0ced5f04ea4968d8cbe16622",
+    "0xba6ad4af6a81370e9a1ba383ce9ecdd3b6131a6a",
+    "0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c",
+    900,
+  ],
+  [
+    "0x1199c95ca9c042773b9a384e5623f8e39aff80c8",
+    "0x148ec100bd2145314628b8be168ab66fb64c639c",
+    "0x2859e4544c4bb03966803b044a93563bd2d0dd4d",
+    1212121,
+  ],
+  [
+    "0x07d7734e8c8bdf1be4e3ba42cb552eefc47168c2",
+    "0x148ec100bd2145314628b8be168ab66fb64c639c",
+    "0x55d398326f99059ff775485246999027b3197955",
+    987659999,
+  ],
+];
+
+//pool, token, balance, block
+const BALANCES: any[][] = [
+  [createAddress("0xc84a"), createAddress("0xbbcd"), BigNumber.from(12432423423), 10001],
+  [createAddress("0xb84a"), createAddress("0x223a"), BigNumber.from(42432423423), 9000],
+  [createAddress("0xa84a"), createAddress("0x5343"), BigNumber.from(52432423423), 12102121],
+  [createAddress("0xd84a"), createAddress("0x271b"), BigNumber.from(62432423423), 9870659999],
+];
+
+describe("PancakeSwap pair fetcher test suite", () => {
```

Comment:
```suggestion
describe("PancakeSwap Data Fetcher Test Suite", () => {
```
Small suggestion to since it is not only fetching the `pair`.